api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/MakePaymentMessageProcessor.java

```diff
@@ -48,16 +48,17 @@ public MessageStatus processMessage(Message message) {
         paymentService.makePaymentForCycle(paymentCycle, messageContext.getCardAccountId());
         EmailMessagePayload messagePayload = buildPaymentNotificationEmailPayload(paymentCycle);
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
-        int numChildrenTurningFourInNextMonth = childDateOfBirthCalculator.getNumberOfChildrenTurningFourAffectingNextPayment(paymentCycle);
-        if (numChildrenTurningFourInNextMonth > 0) {
-            sendChildTurnsFourEmail(paymentCycle, numChildrenTurningFourInNextMonth);
+        NextPaymentCycleSummary dateOfBirthSummaryAffectingNextPayment =
```

Comment:
suggest calling the variable `nextPaymentCycleSummary`

Reply:
Yep, will do in the next PR.